iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -40,6 +40,70 @@ TEST_F(RoudiFindService_test, OfferSingleMethodServiceSingleInstance)
     ASSERT_THAT(*instanceContainer.begin(), Eq(IdString("instance1")));
 }
 
+TEST_F(RoudiFindService_test, OfferServiceWithDefaultDescription)
+{
+    senderRuntime->offerService(iox::capro::ServiceDescription());
+    this->InterOpWait();
+    iox::runtime::InstanceContainer instanceContainer;
+    receiverRuntime->findService(iox::capro::ServiceDescription(), instanceContainer);
+
+    ASSERT_THAT(instanceContainer.size(), Eq(1u));
+    senderRuntime->stopOfferService(iox::capro::ServiceDescription());
+}
+
+TEST_F(RoudiFindService_test, ReOfferSingleMethodServiceSingleInstance)
+{
+    senderRuntime->offerService({"service1", "instance1"});
```

Comment:
Does it make sense to offer a service with an invalid event id?

Reply:
This is some kind of ServiceDescription 'light' which you can use for service discovery in gateways where the protocol does not necessarily need  an event id. According to the implementation of Servicedescription you can construct it with a service and instance id only. In this case is the bool flag m_hasServiceOnlyDescription set. 

From my point of view it is valid to have it because not everyone needs this event id and we give more freedom to the users.